code/_onclick/hud/screentip.dm

```diff
@@ -1,3 +1,5 @@
+GLOBAL_LIST_EMPTY(announcements_huds)
```

Comment:
why do we need more then one of these

Reply:
because they're per HUD?

Reply:
why exactly? we show the same one to everyone no?